### DISCUSSION TITLE
The progress bar is shifted when dragging with a set transform: scale

### DISCUSSION THREAD

**Discussion by dmytro-kuz** [katspaugh/wavesurfer.js#3167] [Q&A] [answered]
Hi, you have a great product wavesurfer.js, but I hope you can help me with this bug that is blocking my work.

## Bug description
When I drag the progress bar in Wavesurfer.js with the enabled 'dragToSeek' option and set 'transform: scale(_scaleFactor_)' to the Wavesurfer container, the progress bar shifts to the left side. When I simply click at any position, it works correctly (default option response: true - works). However, when I start dragging this progress, it shifts further to the left as I increase the scaleFactor.

## Environment
 - Browser: Chrome
 - Operating system: Windows 11

## Expected result
The progress bar when dragging should follow the cursor without shifting to the left side

## Obtained result
Here is a CodePen demo with the reproduced issue: https://codepen.io/Dmytro-Kuz/pen/KKbNwqZ?editors=1111

**Comment by katspaugh** [comment]
That's expected and not a bug. Scaling without setting a transform-origin is done around the center point by default, which shifts everything. I'm surprised the click position is still correct though. 🤔

**Comment by dmytro-kuz** [comment]
Perhaps you can advise me on how I can fix this behavior in my case?

**Reply by katspaugh** [reply, marked as answer]
My bad, it _is_ on wavesurfer.js's side. Here's a fix: https://github.com/katspaugh/wavesurfer.js/pull/3168

**Reply by dmytro-kuz** [reply]
Thanks for such a quick solution

**Reply by katspaugh** [reply]
I've released this fix in 7.2.2.

**Reply by dmytro-kuz** [reply]
thanks, everything works perfect
